bin/test-example

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Travis can help enforce that there are no Bash-specific features in /bin/sh files. /bin/sh is not bash on Travis. I hope it stays that way.